source/HostPhageUnitTest.test.cc

```diff
@@ -230,3 +230,94 @@ TEST_CASE("Host phage death and removal from syms list"){
         }
     }
 }
+
+TEST_CASE("Phage Horizontal Transmission") {
+    emp::Ptr<emp::Random> random = new emp::Random(9);
+    SymWorld w(*random);
+    SymWorld * world = &w;
+    SymConfigBase config;
+
+    config.LYSIS(1); //phage process only happens when lysis is enabled
+    config.GRID_X(2); 
+    config.GRID_Y(1);
+    config.SYM_LIMIT(2);
+    int location = 0;
+
+    WHEN("The phage chooses to Horizontally Transmit"){
+        config.LYSIS_CHANCE(1.0); //100% chance of lysis, 0% chance of lysogeny
+        config.SYM_LYSIS_RES(5);
+        config.BURST_TIME(10);
```

Comment:
This 10 is used in two places, so it'd be better for it to be a variable to make it clear that they are linked